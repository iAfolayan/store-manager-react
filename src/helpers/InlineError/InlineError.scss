.inlineError {
  color: #F00;
  font-size: 1rem;
  text-align: left;
  margin-top: 0.5rem;
  display: block;
}