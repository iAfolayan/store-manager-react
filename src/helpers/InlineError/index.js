import React from 'react';
import PropTypes from 'prop-types';
import './InlineError.scss';

function InlineError({ text }) {
  return <span className="inlineError text-danger">{ text }</span>;
}
InlineError.propTypes = {
  text: PropTypes.string.isRequired
};

export default InlineError;
