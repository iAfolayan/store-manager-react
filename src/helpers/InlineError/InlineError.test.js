import React from 'react';
import { shallow } from 'enzyme';
import InlineError from '.';

describe('Test inlineError Component', () => {
  test('should matches the snapshot', () => {
    const wrapper = shallow(<InlineError text="" />);
    expect(wrapper).toMatchSnapshot();
  });
});
