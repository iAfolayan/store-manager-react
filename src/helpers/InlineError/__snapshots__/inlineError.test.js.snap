// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test inlineError Component should matches the snapshot 1`] = `
<span
  className="inlineError text-danger"
/>
`;
